.docs/user-guide/usage.md

```diff
@@ -0,0 +1,207 @@
+| `-v` | `--verbose=true` | emit verbose information to console |
+
+## Commands
+
+| Command | Description |
+| --- | --- |
+| version | return REX-Ray and libStorage versions |
+| env | return effective current configuration, in form normalized as environment variables |
+| adapter | return list of configured services, or list of host instances manageable by a service |
+| device | list or operate (mount,unmount,format) on OS devices |
+| service | manage REX-Ray running as an OS service |
+| volume | list or operate (create,remove,attach,detach,mount,unmount) on volumes |
+| help | return list of commands and global option flags |
+
+## Examples
+
+**Display Version**
+
+```shell
+$ rexray version
+REX-Ray
+-------
+Binary: /usr/bin/rexray
+Flavor: client+agent+controller
+SemVer: 0.8.1
+OsArch: Linux-x86_64
+Branch: v0.8.1
+Commit: 30e9082dd9917f0d218ca981f886d701110ce5f5
+Formed: Sat, 25 Feb 2017 03:00:28 UTC
+
+libStorage
+----------
+SemVer: 0.5.1
+OsArch: Linux-x86_64
+Branch: v0.8.1
+Commit: 35c7b6d96d5f17aa0c0379924615ae22c1ad3d45
+Formed: Sat, 25 Feb 2017 02:59:00 UTC
+```
+
+**Start REX-Ray Service**
+
+```shell
+$ sudo rexray service start
+Starting REX-Ray...SUCCESS!
+
+  The REX-Ray daemon is now running at PID 1455. To
+  shutdown the daemon execute the following command:
+
+    sudo /usr/bin/rexray stop
+```
+
+**List Volumes**
+
+```shell
+$ rexray volume ls
+ID                     Name       Status       Size
+vol-03ff12d1be6e8a65d             attached     8
+vol-549bd6d4                      unavailable  20
+vol-867e4906           mesos1-sw  available    16
+```
+
+**List Configured Drivers**
+
+```
+$ rexray adapter types
+Name  Driver
+ebs   ebs
+```
+
+**List Host Instances**
+
+List all host instances manageable by configured services.
+
+```shell
+$ rexray adapter instances
+ID                   Name                 Provider  Region
+i-0e46f04c4b7449bc1  i-0e46f04c4b7449bc1  ebs       us-west-2
+```
+
+**Create Volume**
+
+Volume create options. This is a partial list of the most commonly used 
+options. To see the complete list of options, 
+invoke `rexray volume create --help`
+
+!!! note "note"
+
+    REX-Ray passes these options to 
+    [libStorage](http://libstorage.readthedocs.io/en/stable/user-guide/storage-providers/) 
+    which in turn invokes the API of a storage provider. The settings range, 
+    and constraints, are generally determined by the storage platform, and 
+    can have cross-option interactions (e.g. specifying a high iops settings, 
+    reduces the allowed range of size). Cloud providers have been known 
+    to change constraints over time and across regions. Please consult the 
+    documentation associated with the storage provider should you 
+    encounter issues creating a volume. 
+
+| Flag | Range | Description | Mandatory |
+| --- | --- | --- | --- |
+| `--fsType=` | `ext4,xfs` | file system format for block device mount | no, default=ext4 |
```

Comment:
This subset of `create` options is a bit misleading if you dig into it.  Both `--fsType` and `--overwriteFS` don't usually do anything with `create`, they are meant for `mount`. However, the `create` command allows for `--attach` and `--amount` options. The former attaches after creation, the latter attaches and mounts. If using `--attach`, you can also specify `--force` (this is requesting pre-emption). If using `--amount` you can specify `--force`, `--fsType`, and `--overWriteFS`.

This is a bit of advanced usage, so I would probably leave those out of this table to not confuse people.